packages/syntax-core/src/Typography/Typography.module.css

```diff
@@ -229,3 +229,8 @@
 .preLine {
   white-space: pre-line;
 }
+
+:lang(ko),
+.typography {
```

Comment:
@somethiiing Did you check whether this is correct? I noticed that all text now has `word-break: keep-all`, not just Korean.

```suggestion
/* stylelint-disable-next-line selector-max-compound-selectors -- https://github.com/w3c/csswg-drafts/issues/4285 */
:lang(ko) .typography {
  word-break: keep-all;
}
```

Also, I would move these lines to where we have the other language specific overrides:
https://github.com/Cambly/syntax/blob/76d59226915367c84f9ccdf0497de9911c146398/packages/syntax-core/src/Typography/Typography.module.css#L57-L58

![image](https://github.com/user-attachments/assets/d49458c2-3daf-4540-9d0a-9c470b21e335)